.github/workflows/test-environment.yml

```diff
@@ -152,6 +152,18 @@ jobs:
           echo "::add-mask::$ec_api_key"
           echo "TF_VAR_ec_api_key=$ec_api_key" >> $GITHUB_ENV
 
+      - name: Process BC version
+        id: remove-commit-hash
+        run: |
+          # Extract the stack version
+          stack_version="${{ inputs.elk-stack-version }}"
+
+          # Check if the version contains a commit hash, remove it
+          if [[ $stack_version =~ -[a-f0-9]+ ]]; then
```

Comment:
I see that we also clean that on the terraform? Where do we use the full version?

Reply:
For Terraform, we are utilizing the `TF_VAR_stack_version: ${{ inputs.elk-stack-version }} `environment variable, which is passed as is.